central/policy/search/searcher_impl.go

```diff
@@ -104,6 +105,7 @@ func convertPolicy(policy *storage.Policy, result search.Result) *v1.SearchResul
 func formatSearcher(unsafeSearcher blevesearch.UnsafeSearcher) search.Searcher {
 	safeSearcher := blevesearch.WrapUnsafeSearcherAsSearcher(unsafeSearcher)
 	transformedSortFieldSearcher := sortfields.TransformSortFields(safeSearcher, policyMapping.OptionsMap)
-	paginatedSearcher := paginated.Paginated(transformedSortFieldSearcher)
+	transformedCategoryNameSearcher := policycategory.TransformCategoryNameFields(transformedSortFieldSearcher)
```

Comment:
This transformation must be done behind feature flag.

Reply:
Check transform func - it returns if feature flag is not set.